git-gutter.el

```diff
@@ -951,13 +963,16 @@ start revision."
     (with-temp-file tmpfile
       (insert content))))
 
-(defsubst git-gutter:original-file-content (file)
+(defsubst git-gutter:original-file-content (file repo-dir)
   (with-temp-buffer
-    (when (zerop (process-file "git" nil t nil "show" (concat ":" file)))
+    (when (zerop (process-file
+                  "git" nil t nil "show"
+                  (concat ":0:" (file-relative-name file repo-dir))))
       (buffer-substring-no-properties (point-min) (point-max)))))
 
 (defun git-gutter:write-original-content (tmpfile filename)
```

Comment:
This calls `git-gutter:original-file-content` using the git root directory stored in `git-gutter:repo-base`.